nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/AbstractBlobProvider.java

```diff
@@ -53,4 +53,12 @@ public boolean supportsUserUpdate() {
         return supportsUserUpdateDefaultTrue();
     }
 
+    /**
+     * @since 8.4
```

Comment:
No need for `@since` Javadoc on the implementation as it's already on the interface.